observation-service/src/main/java/gov/cdc/etldatapipeline/observation/repository/model/reporting/ObservationNumeric.java

```diff
@@ -0,0 +1,26 @@
+package gov.cdc.etldatapipeline.observation.repository.model.reporting;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class ObservationNumeric {
+    private Long observationUid;
+    private String ovnHighRange;
+    private String ovnLowRange;
+    @JsonProperty("ovn_comparator_cd_1")
+    private String ovnComparatorCd1;
+    @JsonProperty("ovn_numeric_value_1")
+    private BigDecimal ovnNumericValue1;
```

Comment:
Does big decimal sink into SQL Server?

Reply:
I googled it, like the "Java type for SQL Server numeric" - it returns `BigDecimal`.
[Java and SQL Server supported data types](https://learn.microsoft.com/en-us/sql/language-extensions/how-to/java-to-sql-data-types?view=sql-server-ver16)

Reply:
It's moreso how the sink maps it. We haven't tried BigDecimal yet, so it might be the correct mapping. Date had an issue, so the String data type assignment works best for it.

Reply:
Agree, at least in the topic message it looks fine now:
```
  "payload": {
    "observation_uid": 10312596,
    "ovn_high_range": "100",
    "ovn_low_range": "0",
    "ovn_comparator_cd_1": "=",
    "ovn_numeric_value_1": 1,
    "ovn_numeric_value_2": null,
    "ovn_numeric_unit_cd": "mL",
    "ovn_separator_cd": null
  }
  ```